hardhat.config.ts

```diff
@@ -46,6 +47,10 @@ const config: HardhatUserConfig = {
         : undefined,
     },
   },
+  tenderly: {
+    username: "thesis",
+    project: "keep-test", // hardhat-tenderly plugin is used on ropsten only
+  },
```

Comment:
Project name was removed here: 681c438

Reply:
We could work around a missing feature by setting the project slug as env variable.
```ts
  tenderly: {
    username: process.env.TENDERLY_USERNAME,
    // Workaround for plugin not being able to read configuration
    // from a YAML file.
    // https://github.com/Tenderly/hardhat-tenderly/issues/17
    project: process.env.TENDERLY_PROJECT_SLUG || "thesis/keep",
  },
```

Reply:
The thing is that no matter what we put here, it won't be used during the contracts `push/verify` to Tenderly. It will produce a similar link to this one `https://dashboard.tenderly.co/contract/ropsten/0x1e83039a7f14d035ec0de7cfd9044a24860ce14a` Note `.../contract/ropsten/...` in url vs. `.../<username>/<project>/...` from `tenderly` config. This is a disadvantage compared to `tenderly_cli` which can put contracts under the specified user's project. Not sure if this is a bug or it is supposed to be like this. I [asked](https://discord.com/channels/511504443653750785/511508648334196747/880468136627933195) this question on Discord, but got no answer. Anyway, I can put it in here, but it won't be used to put contracts under the right project, this step will need to be done manually - [Moving contracts to the right Tenderly project](https://coda.io/d/Building-Keep_d-fmEgBNFVH/Current-CI-process_su1ww#_lunBK).

Reply:
> The thing is that no matter what we put here, it won't be used during the contracts push/verify to Tenderly.
> Not sure if this is a bug or it is supposed to be like this. I asked this question on Discord, but got no answer. 

😞  We should create a GitHub issue and link it in the code comment.

Reply:
Issue is created https://github.com/Tenderly/hardhat-tenderly/issues/20